components/resources/BUILD.gn

```diff
@@ -107,6 +107,10 @@ action("about_credits") {
     "--target-os=$target_os",
     "--depfile",
     rebase_path(depfile, root_build_dir),
+    "--gn-out-dir",
+    ".",
+    "--gn-target",
+    "//chrome", # TODO(toshok) can't imagine we want this, but I'm not sure how else to do it?
```

Comment:
this is probably the sketchiest part of these changes.  Is there any way to figure out which toplevel target we're building?

Reply:
I do not know, but `//chrome` seems like a safe bet for us?

Reply:
yeah, definitely my feeling